slack-market/api/endpoints/slack_endpoint.rb

```diff
@@ -0,0 +1,48 @@
+require 'pp'
+module Api
+  module Endpoints
+    class GraphEndpoint < Grape::API
+      format :json
+
+      namespace :graph do
+        desc 'Select graph.'
+
+        params do
+          requires :payload, type: String
+        end
+
+        post do
+          p params.class
+          # slack interactive messages send a payload.
```

Comment:
Maybe just replace this comment with a link to slack doc or something, it's pretty clear what the code does here.